image-sync/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM python:3-slim AS build
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# hadolint ignore=DL3008
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+               build-essential gcc git \
```

Comment:
Do we really need `build-essentials` and `gcc` as dependencies?

Reply:
Changed to regular python image instead of slim so no need to install these